app/src/main/java/pl/droidevs/books/model/Book.java

```diff
@@ -0,0 +1,36 @@
+package pl.droidevs.books.model;
+
+public class Book {
```

Comment:
Myślę, że w przyszłości będzięmy dążyć, aby Book domenowy był obiektem, a nie strukturą danych.

Reply:
jasne :) skor approved - to merguje! :)

Swoją drogą github mógłby odświeżać się automatycznie z nowymi wiaodmościami :(